Draft: Add options to start and stop group replication

eryx12o45 opened the pull request:
##### SUMMARY
- Added options to start and stop group replication

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
mysql_replication moodule

##### ADDITIONAL INFORMATION

Andersson007 commented:
@eryx12o45 hi, thanks for the PR. It's a draft but anyway, please
1) add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to)
2) add tests to make CI green
3) maybe it's worth adding underscores to the new values to make them more human friendly?

Thanks!

eryx12o45 commented:
@Andersson007 thanks for the comments. I've added a changelog fragment and renamed the commands. Unfortunately I will not be able to provide tests in the near future, because of time constraints and the fact, that I would need to extend the setup routine to add group replication configs, etc. otherwise the commands would never work :-(

Andersson007 commented on changelogs/fragments/mysql_replication_add_group_replication_commands.yml:
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - mysql_replication - added options to `start_group_replication` and `stop_group_replication`
```
```suggestion
  - mysql_replication - add ``start_group_replication`` and ``stop_group_replication`` options (https://github.com/ansible-collections/community.mysql/pull/93).
```

Andersson007 reviewed (commented):
@eryx12o45 thanks for fixing! I'll take a deeper look when time allows

Andersson007 commented:
@betanummeric @laurent-indermuehle @bmalynovytch @rsicart @Jorge-Rodriguez what do you think about these changes? Can we merge it w/o tests? (provided that the fragment is fixed)

Jorge-Rodriguez commented:
This PR needs rebase. I would wait for #63 to be merged and then rework these changes on top of it. Then we could add tests as well.

Andersson007 commented:
@eryx12o45 could you please [rebase](https://docs.ansible.com/ansible/latest/dev_guide/developing_rebasing.html) the PR? if you have no time, let us know

eryx12o45 commented:
New version here --> https://github.com/ansible-collections/community.mysql/pull/647
